packages/accordions/src/utils/scroll.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+export function disableScroll() {
+  document.body.style.overflow = 'hidden';
+
+  const x = window.scrollX;
+  const y = window.scrollY;
+
+  window.onscroll = () => {
+    window.scrollTo(x, y);
+  };
+}
+
+export const enableScroll = () => {
+  setTimeout(() => {
+    // Set overflow on the next tick of event loop so Chromium does not add scrollbars.
+    document.body.style.overflow = '';
```

Comment:
I'll leave it to further review for comments on the overall approach, but don't we commit to pulling `document` off of the current theme? https://github.com/zendeskgarden/react-components/blob/main/packages/theming/src/utils/getDocument.ts

Reply:
We provide a `useDocument` hook that can be used with the `ThemeContext` provided by styled-components.